docs/contributing/feature-flags.md

```diff
@@ -20,6 +20,65 @@ highlights:
 - Environments (production, QA, and development for now) exist to segment flag states further. This
   will be automatic based on where code is running.
 
+## When to use flags
+
+Feature flags are a powerful tool for controlling the rollout and lifecycle of features, and there
+are several scenarios when you should consider using them:
```

Comment:
Do we want a stronger statement here - e.g. the default should be to feature flag all material changes unless there's a reason not to?